backend/workflow_manager/endpoint/database_utils.py

```diff
@@ -454,3 +483,25 @@ def python_to_bigquery_mapping(python_type: Any) -> Optional[str]:
             datetime.datetime: "TIMESTAMP",
         }
         return mapping.get(python_type, "string")
+
+    @staticmethod
+    def to_redshift_mapping(python_type: Any) -> Optional[str]:
+        """Method to convert python to datatype Used by Redshift."""
+        mapping = {
+            str: "VARCHAR(65535)",
```

Comment:
`65535` Any reason for this value?

Reply:
@muhammad-ali-e redshift holds max upto `65535` character